joomla/joomla-cms
# [4] com_modules empty state

Can display for Site and Administrator Modules (Only for completeness, as near impossible to replicate for Admin Modules Empty State) 

### Testing Instructions

Trash the three modules Joomla 4 installed when you installed Joomla. Empty the trash. See the Empty State screen. Use the buttons to create a new Site Module, see that it works as expected 

For Administrator Modules - code review only ;-) 



<img width="1213" alt="Screenshot 2021-04-28 at 22 52 32" src="https://user-images.githubusercontent.com/400092/116477387-71920600-a874-11eb-8153-e42d6d8f0437.png">
<img width="1023" alt="Screenshot 2021-04-28 at 22 52 53" src="https://user-images.githubusercontent.com/400092/116477410-7a82d780-a874-11eb-8639-cd0c5f6b6544.png">
